srcpkgs/docker-cli/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'docker-cli'
+pkgname=docker-cli
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/docker/cli"
+hostmakedepends="pkg-config tar"
+depends="containerd runc iptables xz git"
```

Comment:
Is there really a hard run-time dependency on `git`? Should `docker-cli` depend on these backends, or is it more appropriate to depend on `moby`? (I use Docker very passively and am not sure how `docker-cli` would function without the `moby` engine.)

Even if depending on `containerd` instead of `moby` is appropriate, you can drop the `runc` dependency because `containerd` depends on `runc`.

Reply:
The docker cli is capable of communicating with any docker daemon, remotely or locally. This is why I set the package up as a meta, rather than strict dependencies. It is also practical in some cases to use the docker daemon without the docker CLI, such as when using a different device to control the daemon.

Reply:
Not depending on moby seems appropriate then. It also sounds like `containerd` and `runc` (and maybe `iptables?) should not be dependencies of `docker-cli` because people may wish to install `docker-cli` for controlling remote moby instances without pulling in underlying container runtimes.

Reply:
See my larger response.